### PR TITLE
Hireling fix 1

### PR DESCRIPTION
**Pull request by Blogaugis**
Added a "Combat Knife" to Skitarii soldiers;
Fixed Inquisition Crusader issue with gear item slot having an item belonging in a weapon group;
Added "Ork Armour" to "Ork Sniper";
Lots of TODO comments added;
Fixes the Crusaders not having Armour

### CODE REVIEW

File: scripts/scr_add_man/scr_add_man.gml [hunk 1]
```diff
@@ -35,12 +35,63 @@ function scr_add_man(man_role, target_company, spawn_exp, spawn_name, corruption
 		if (other_gear = true) {
 	// Factions 1-5 are part of Imperial family
 		// Faction 1 - Space Marine
+			/* TODO - add the right faction stuff, and move the "switch (man_role) {" before uncommenting
```
Comment (OH296) [inline scripts/scr_add_man/scr_add_man.gml:38]: we will not be making new additions using the scr_add_man script like this in the furture it's terrible practise, but these can stay commented for now as guidane

File: scripts/scr_add_man/scr_add_man.gml [hunk 2]
```diff
@@ -9,7 +9,7 @@ function scr_add_man(man_role, target_company, spawn_exp, spawn_name, corruption
 
 	// That should be sufficient to add stuff in a highly modifiable fashion
 
-	var non_marine_roles = ["Skitarii", "Techpriest", "Crusader", "Sister of Battle", "Sister Hospitaler", "Ranger", "Ork Sniper", "Flash Git"];
+	var non_marine_roles = ["Skitarii", "Techpriest", "Crusader", "Sister of Battle", "Sister Hospitaler", "Ranger", "Ork Sniper", "Flash Git"]; // if adding new hirelings, don't forget to update this list
```
Comment (OH296) [inline scripts/scr_add_man/scr_add_man.gml:12]: this list will be irrelevant soon this is just retrospective of the old code base

**Review by OH296 — CHANGES_REQUESTED** [review]
I would suggest you familiarise yourself with the concept of OOP and the scr_marine_struct rather than focussing on scr_add_man its a poorly written file that we will probably not continue to expand and support

File: scripts/scr_add_man/scr_add_man.gml (hunk 1 above)
Comment (Blogaugis) [reply scripts/scr_add_man/scr_add_man.gml:38]: Ok, I'll try fiddling around with the scr_marine_struct in future PRs.

File: scripts/scr_add_man/scr_add_man.gml (hunk 2 above)
Comment (Blogaugis) [reply scripts/scr_add_man/scr_add_man.gml:12]: Understood.

**Review by EttyKitty — COMMENTED** [review]
I think considering that https://github.com/EttyKitty/ChapterMaster/pull/27 already adds light power armour and light bolter, you may want to merge loadout bits of this PR into that PR instead, and close this one.

**Comment by Blogaugis** [comment]
Understood, I'll try to merge it to #27 .

**Comment by Blogaugis** [comment]
Done, it seems.

**Pull request closed without merging**